README.md

```diff
@@ -1,2 +1,24 @@
-# ms-office-macro-nuke
-Tool to Purge and Remove all Macro And Dynamic Content from MS Office File
+![MS Office Macro Bleach](docs/images/bleach.jpg)
+
+
+# MS Office Macro Bleach
+
+A command-line tool designed to detect and purge any and all macros and dynamic content from MS Office files.
```

Comment:
You could change the wording to "commonly used office document formats (including MS Office Files, PDFs etc).